src/components/HelpMenu/index.tsx

```diff
@@ -12,24 +12,26 @@ import {
 import { ButtonClass } from "../../constants/interfaces";
 import { DownArrow } from "../Icons";
 import VersionModal from "../VersionModal";
-import NavButton from "../NavButton";
 import CustomDropdown from "../CustomDropdown";
+import {
+    DropdownAnchor,
+    DropdownButton,
+    DropdownRouterLink,
+} from "../CustomDropdown/DropdownMenuItems";
 
 const HelpMenu = (): JSX.Element => {
     const [modalVisible, setModalVisible] = React.useState(false);
 
     const location = useLocation();
     const tutorialLink =
         location.pathname === "/viewer" ? (
-            <Link
-                to={TUTORIAL_PATHNAME}
-                target="_blank"
-                rel="noopener noreferrer"
-            >
+            <DropdownRouterLink to={TUTORIAL_PATHNAME} newTab={true}>
                 Quick start
-            </Link>
+            </DropdownRouterLink>
         ) : (
-            <Link to={TUTORIAL_PATHNAME}>Quick start</Link>
+            <DropdownRouterLink key="Tutorial" to={TUTORIAL_PATHNAME}>
```

Comment:
Why does this have a key when the other element doesn't? Should they both have keys?

Reply:
Not sure why I was inconsistent here, but in taking a look at it I think they aren't really needed, I could add where missing, but might also just remove most of them.

These components, if they were directly populating an array would each need keys, but the components are the values of the `label` properties, not directly filling the arrays so I think it's ok to just remove?